nucypher/config/base.py

```diff
@@ -799,9 +799,15 @@ def write_keystore(self, password: str, key_material: Optional[bytes] = None, in
                                                      password=password,
                                                      keystore_dir=self.keystore_dir)
         else:
-            self.__keystore = Keystore.generate(password=password,
-                                                keystore_dir=self.keystore_dir,
-                                                interactive=interactive)
+            if interactive:
+                self.__keystore = Keystore.generate(password=password,
+                                                    keystore_dir=self.keystore_dir,
+                                                    interactive=interactive)
+            else:
+                self.__keystore, _ = Keystore.generate(password=password,
```

Comment:
`_` is unused here, and `self.keystore` is returned in both cases.  Is this the intended behavior?

Reply:
@derekpierre - The difference here is whether or not to unpack the result, assigning `self.__keystore` to a tuple or a keystore instance..

Reply:
Ah so we had it flipped, they care about the words when interactive is enabled; so line 803.

👍